src/test/java/odysseusAT/CharacterizationStepDefs.java

```diff
@@ -0,0 +1,383 @@
+package odysseusAT;
```

Comment:
Rename core package to AtlasTests (`atlastests`)

Reply:
done